modules/nw-sriov-network-attachment.adoc

```diff
@@ -28,20 +28,35 @@ apiVersion: sriovnetwork.openshift.io/v1
 kind: SriovNetwork
 metadata:
   name: <name> <1>
-  namespace: sriov-network-operator <2>
+  namespace: openshift-sriov-network-operator <2>
 spec:
   networkNamespace: <target_namespace> <3>
   ipam: |- <4>
     ...
   vlan: <vlan> <5>
   resourceName: <sriov_resource_name> <6>
+  linkState: <link_state> <7>
+  maxTxRate: <max_tx_rate> <8>
+  minTxRate: <mix_rx_rate> <9>
+  vlanQoS: <vlan_qos> <10>
+  spoofChk: <spoof_check> <11>
+  trust: <trust_vf> <12>
+  capabilities: <capabilities> <13>
+  
 ----
 <1> Replace `<name>` with a name for the CR. The Operator will create a NetworkAttachmentDefinition CR with same name.
 <2> Specify the namespace where the SR-IOV Operator is installed.
-<3> Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR will be created.
-<4> Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
-<5> Replace `<vlan>` with a Virtual LAN (VLAN) ID for the additional network. The integer value must be from `0` to `4095`. The default value is `0`.
+<3> Optional. Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR will be created. The default value is `openshift-sriov-network-operator`.
+<4> Optional. Specify a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<5> Optional. Replace `<vlan>` with a Virtual LAN (VLAN) ID for the additional network. The integer value must be from `0` to `4095`. The default value is `0`.
 <6> Replace `<sriov_resource_name>` with the value for the `.spec.resourceName` parameter from the SriovNetworkNodePolicy CR that defines the SR-IOV hardware for this additional network.
+<7> Optional. Replace `<link_state>` with a the link state of VF. Allowed value are `enable`, `disable` and `auto`.
+<8> Optional. Replace `<max_tx_rate>` with a Maximum tx rate, in Mbps, for the VF.
+<9> Optional. Replace `<min_tx_rate>`` with a Minimum tx rate, in Mbps, for the VF. This value should always be less than or equal to Maximum tx rate. Note: Intel NICs don't support this parameter.
+<10> Optional. Replace `<vlan_qos>` with a IEEE 802.1p Priority levels for the VF. The default value is `0`.
+<11> Optional. Replace `<spoof_check>` with the spoof check mode of the VF. Allowed values are `"on"` and `"off"`. 
+<12> Optional. Replace `<trust_vf>` with the trust mode of the VF. Allowed values are `"on"` and `"off"`. If `on` or `off` is specified without quotes, it will be rejected by CRD validation, since it will be evaluated as boolean value.  
```

Comment:
@pliurh -

 it sounds like `trust` is treated differently if it's specified as `trust: "on"` versus `trust: on`, but I thought in YAML, a flow scalar is equivalent whether it's quoted or not in the simplest cases? Does Go handle YAML differently?

(http://blogs.perl.org/users/tinita/2018/03/strings-in-yaml---to-quote-or-not-to-quote.html)

Thanks!

cc @zshi-redhat

Reply:
@jboxman It should be the same as spoof_check.

Reply:
I suggested to use quotes. otherwise it will report error when creating sriov network according to previous tests.

Reply:
It looks like go-yaml tend to support both 1.1 and 1.2. some relevant readings:
https://github.com/kubernetes/kubernetes/issues/34146
https://github.com/go-yaml/yaml/issues/214